src/main/java/edition/academy/seventh/serivce/EmpikScrapping.java

```diff
@@ -0,0 +1,89 @@
+package edition.academy.seventh.serivce;
+
+import edition.academy.seventh.database.model.Book;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Phaser;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+/**
+ * Scrap data from empik.com bookstore website in sales section using JSoup library.
+ *
+ * @author Bartosz Kupajski
+ */
+public class EmpikScrapping implements IPromotionScrapping {
+
+    private List<Book> listOfBooks = new CopyOnWriteArrayList<>();
+    private ExecutorService service = Executors.newFixedThreadPool(40);
+    private Phaser phaser = new Phaser(1);
+  /**
+        Above that number SSLException is thrown
+     */
+    private final int maxConnections = 1201;
+
+  /**
+   * Index in for loop is incremented by 30 because of the fact that
```

Comment:
Ojojojojoj no weźcie ludki

Co robi ta metoda???
fajnie że jest dokumentacja do tego jak działa ta metoda ale głównym jej celem powinno być tłumaczenie co ona robi a nie jak

Index inside "for" loop is incremented by 30 since every URL page of empik's sales section contains id which is multiplicity of 30.